notebooks/7-rotation-speeds/interm_rotation_rate.ipynb

```diff
@@ -0,0 +1,2926 @@
+{
```

Comment:
<p>Add something like: "this is related to/based on Fourier transforms" [link to resource on Fourier] <a href="https://www.3blue1brown.com/lessons/fourier-transforms" target="_blank">fourier-transforms</a>. 3b1b is maybe too in-depth, but the wikipedia article is kind of intimidating</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>